libcaf_net/caf/net/transport_base.hpp

```diff
@@ -225,7 +223,7 @@ class transport_base {
   buffer_cache_type header_bufs_;
   buffer_cache_type payload_bufs_;
 
-  buffer_type read_buf_;
+  byte_buffer read_buf_;
```

Comment:
`caf/byte_buffer.hpp` is not included in this file.

Reply:
Same here.